website/allowed-subcategories.txt

```diff
@@ -27,6 +27,7 @@ Key Management Service
 Kubernetes Service
 Logs Router
 Metrics Router
+MQ on Cloud
```

Comment:
Is this same name refered generally on your cli,api ,cloud docs?
Do we need to mention explictly "on Cloud"

Reply:
Yes, we need to explicitly mention 'on Cloud' here, as both MQ and MQ on Cloud are separate services.

In the apidocs and sdk we are following this name itself 
- api file: https://github.ibm.com/cloud-api-docs/mq-on-cloud/blob/6840e3fcdfea5cf6ebf9f2a742d3fc98da288cca/mq-on-cloud.yaml#L4
- sdk: https://github.com/IBM/mqcloud-go-sdk/#ibm-cloud-mqoc-services-go-sdk-001

Reply:
Hey @hkantare 

After discussing with the team about the name we should use, we encountered some conflicts regarding the product name. Here is the full list of names we have been using:

1. `MQ` - used in the IBM Cloud catalog (UI)
2. `mqcloud` - registered service name in RMC
3. `MQ on Cloud` - used in the doc and APIs, SDKs
4. `IBM MQ on Cloud` - used in docs
5. `IBM MQ on IBM Cloud` - Official registered name of current service
6. `IBM MQ on IBM Cloud Reserved Instance` - Official name of BNPP offering

So which name would you suggest us to use?

Reply:
After some more discussions. we are right now sticking with `MQ on Cloud` itself 👍